dist/tools/pic32prog/doc.md

```diff
@@ -0,0 +1,160 @@
+Flashing on Linux using pickit-3
+================================
+
+This document describes how to enable flashing a pic32-wifire from Linux with a
+PICkit3 using [pic32prog](https://github.com/sergev/pic32prog).
+
+It will require flashing a specific firmware on the PICkit3.
+As this can only be done from a Windows computer, that not many Linux users
+have, the following steps explain how to setup a Windows VirtualBox virtual
+machine and flash the PICkit3 from it.
+
+Informations come from this comment
+
+https://github.com/RIOT-OS/RIOT/pull/6092#issuecomment-261987955
+
+
+Steps
+-----
+
+### Setup your computer
+
+I ran this on Ubuntu 16.04
+
+* Install Virtualbox
+* Add yourself to the 'vboxusers' group
+  `sudo usermod -a -G vboxusers your_username`
+* Reboot
+
+### Setup Windows VM
+
+You can download a free 90 days valid windows virtualbox image from:
+
+    https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/
+
+I downloaded a Windows 10 virtualbox image
+
+* Create a machine in virtualbox in File-Import Appliance.
+* Start the virtual machine and wait until you are on the windows desktop
+* Install Virtualbox Guest-additions for USB support
+  * Doc: https://www.virtualbox.org/manual/ch04.html#additions-windows
+* Shutdown the virtual machine
+* Plug the PICkit3 flasher USB to your computer
+* In the virtual machine settings, go to USB:
+  * Select "Enable USB Controller" with "USB 1.1"
+  * Click the small '+' sign on the right and add the Microchip devices.
+  * Doc: https://www.virtualbox.org/manual/ch03.html#idm1640
+* Start the virtual machine
+
+### Downloads
+
+Download and extract:
+
+* PICkit3-Programmer http://ww1.microchip.com/downloads/en/DeviceDoc/PICkit3%20Programmer%20Application%20v3.10.zip
+
+Adapted from the README (on Windows 10 machine .NET 4.00 is already installed)
```

Comment:
The context of this sentence isn't clear to me (maybe because it is missing a subject or a colon at the end?)